int64 tick_id
int64 flower_id
int64 current_reward
